scripts/enum/item.lua

```diff
@@ -864,6 +864,7 @@ xi.item =
     CATHEDRAL_TAPESTRY                  = 1662,
     CLUSTER_CORE                        = 1667,
     CLEAN_SNAP_ROD                      = 1668,
+    PINCH_OF_HOARY_BOMB_ASH             = 1669,
```

Comment:
This is unused, but oh well...

Reply:
Ah yea - i was working on another quest and apparently it slipped in there. It's coming ;)